src/main/java/io/leonis/subra/game/data/TeamColor.java

```diff
@@ -0,0 +1,50 @@
+package io.leonis.subra.game.data;
+
+import java.awt.Color;
+
+/**
+ * The Enum TeamColor.
+ *
+ * This enumeration represents the possible colors a team can sport in a Small Size League
+ * sequence.
+ *
+ * @author Jeroen de Jong
+ * @author Rimon Oz
+ */
+public enum TeamColor {
+  /**
+   * Blue team color.
+   */
+  BLUE(Color.BLUE),
+  /**
+   * Yellow team color.
+   */
+  YELLOW(Color.YELLOW),
+  /**
+   * None team color.
+   */
+  NONE(Color.BLACK);
```

Comment:
Is this used somewhere?

Reply:
see `GoalsDeducer`. it's used as a fallback when no team color is present. perhaps `GoalsDeducer` can be refactored to get rid of the fallback